### PR TITLE
refactor: Remove state ifo

### PR DESCRIPTION
**Pull request by ChefMomota**

### CODE REVIEW

File: src/views/Ifos/components/IfoPoolVaultCard.tsx [hunk 1]
```diff
@@ -60,107 +58,107 @@ const StyledCardBody = styled(CardBody)`
...
+  // const { pool } = useIfoWithApr()
+  // const { isMd, isXs, isSm } = useMatchBreakpoints()
+  // const isSmallerThanTablet = isMd || isXs || isSm
+  // if (isSmallerThanTablet) {
+  //   return <IfoPoolVaultCardMobile />
+  // }
 
-  return <CakeVaultCard pool={pool} showStakedOnly={false} m="auto" />
+  // return <CakeVaultCard pool={pool} showStakedOnly={false} m="auto" />
+  return <>New Layout</>
```
Comment (ChefMomota) [inline src/views/Ifos/components/IfoPoolVaultCard.tsx]: This component will replace with new layout.

**Comment by ChefMomota** [comment]
Don't merge, still waiting for new ifo page layout.

File: src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx [hunk 2]
```diff
@@ -186,10 +183,7 @@ const IfoCardTokens: React.FC<IfoCardTokensProps> = ({
         </Box>
       ) : null
 
-    if (
-      (ifo.version === 3 || (ifo.version === 3.1 && poolId === PoolIds.poolUnlimited)) &&
-      getBalanceNumber(credit) === 0
-    ) {
+    if (ifo.version === 3 || (ifo.version === 3.1 && poolId === PoolIds.poolUnlimited)) {
```
Comment (0xjojoex) [inline src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx:188]: can you add a comment here, we'll definitely need to add checking in the future

Comment (ChefMomota) [reply src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx:188]: comment added

**Pull request merged**